src/main/resources/templates/tools.html

```diff
@@ -46,14 +46,13 @@ <h4 align="left"><strong>Software-defined Networking (SDN) Provisioning Tools</s
                             <li>The SDN technology rely on the hardware features on switches. Such SDN switches on large networks are not easily available.</li>
                             <li>NCL have 14 SDN switches connecting 300 nodes. </li>
                         </ul>
-                        <p align="left"><strong>Potential Users</strong></p>
-                        <ul>
-                            <li>SDN researchers in academia</li>
-                        </ul>
+                        <!--<p align="left"><strong>Potential Users</strong></p>-->
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)